core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -94,6 +95,7 @@ case class WorkspaceBillingAccount(
 case class RawlsBillingProjectOrganization(enterprise: Boolean, limits: Map[String, String])
 
 case class RawlsBillingProjectResponse(
+  id: String,
```

Comment:
`UUID` instead of `String`?

Reply:
Fixed: https://github.com/broadinstitute/rawls/pull/3205/commits/7d070f460201a426a5fdb8320f42e6293e82d8cc